app/models/invoice.rb

```diff
@@ -4,9 +4,12 @@ class Invoice < ApplicationRecord
   enum status: ["in progress", "completed", "cancelled"]
   has_many :invoice_items
   has_many :items, through: :invoice_items
-  has_many :transactions
```

Comment:
Was removing this just a cleanup thing because it wasn't being used anywhere, or was it necessary to make invoice_items function properly?